catboost/dotnet/CatBoostNet/CatBoostModel.cs

```diff
@@ -31,7 +31,7 @@ public class CatBoostValuePrediction
     /// If you need to implement a custom model code, use <see cref="CatBoostModelEvaluator"/>
     /// as the basic building block.
     /// </summary>
-    public class CatBoostModel : ITransformer
+    public class CatBoostModel : ITransformer, IDisposable
```

Comment:
you inherit from IDisposable but do not write Dispose method

Reply:
Discussed